opmd_viewer/openpmd_timeseries/main.py

```diff
@@ -460,7 +459,7 @@ def get_field(self, field=None, coord=None, t=None, iteration=None,
             field_path = field
             field_label = field
         elif self.fields_metadata[field]['type'] == 'vector':
-            field_path = os.path.join(field, coord)
+            field_path = field + '/' + coord
```

Comment:
Could you also use the new function `filepath_join` here?

Reply:
done!